Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -260,11 +260,13 @@
         Blunt: 0.6
         Slash: 0.6
         Piercing: 0.6
-        Heat: 0.8
+        Heat: 0.8 # Harmony Change - adds resistence (0 -> 0.8)
```

minor spelling misteke
```suggestion
        Heat: 0.8 # Harmony Change - adds resistance (0 -> 0.8)
```